sdk/storage/Azure.Storage.Blobs/src/Generated/Models/BlobBlock.cs

```diff
@@ -14,22 +14,17 @@ namespace Azure.Storage.Blobs.Models
     {
         /// <summary> Initializes a new instance of BlobBlock. </summary>
         /// <param name="name"> The base64 encoded block ID. </param>
-        /// <param name="size"> The block size in bytes. </param>
+        /// <param name="sizeLong"> The block size in bytes. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="name"/> is null. </exception>
-        internal BlobBlock(string name, int size)
+        internal BlobBlock(string name, long sizeLong)
```

Comment:
there should be some code in ModelFactory for this thing. plz make sure it can take long as well.